setup.py

```diff
@@ -28,7 +28,6 @@ def readme():
         'hypothesis',
         'pycodestyle',
         'pylint>=2.1,<2.2',
-        'nose',
```

Comment:
This is good, but you need to add `pytest` to replace it. I *think* it's possible to add this under `extras_require['dev']`, but try it out.